### ISSUE TITLE
Java NoSuchMethodError on SBT 0.13.9

### ISSUE THREAD

**Issue by dmurvihill** [playframework/twirl#187]
Create the following files in a new directory:

project/build.properties:
```sbt.version=0.13.9```

project/plugins.sbt:
```addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.3.15")```

project/Build.scala:
```scala
import sbt.{Project, Build}
import play.twirl.sbt.SbtTwirl

object MyBuild extends Build {
  lazy val mainProject = Project ("myproject", file ("."))
    .enablePlugins(SbtTwirl)
}
```

Then run:
```bash
sbt
```
(Java 8 JDK)

Output:
```[info] Loading global plugins from /Users/dolan/.sbt/0.13/plugins
[info] Loading project definition from /Users/dolan/sbt-twirl-tests/project
[info] Compiling 1 Scala source to /Users/dolan/sbt-twirl-tests/project/target/scala-2.10/sbt-0.13/classes...
java.lang.NoSuchMethodError: sbt.Def$.settings(Lscala/collection/Seq;)Lscala/collection/Seq;
	at play.twirl.sbt.SbtTwirl$.dependencySettings(SbtTwirl.scala:75)
	at play.twirl.sbt.SbtTwirl$.projectSettings(SbtTwirl.scala:41)
	at sbt.Load$$anonfun$autoPluginSettings$1$1.apply(Load.scala:666)
	at sbt.Load$$anonfun$autoPluginSettings$1$1.apply(Load.scala:666)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:251)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:105)
	at sbt.Load$.autoPluginSettings$1(Load.scala:666)
	at sbt.Load$.sbt$Load$$expandSettings$1(Load.scala:681)
	at sbt.Load$$anonfun$sbt$Load$$expandSettings$1$2.apply(Load.scala:682)
	at sbt.Load$$anonfun$sbt$Load$$expandSettings$1$2.apply(Load.scala:682)
	at scala.collection.IndexedSeqOptimized$class.foldl(IndexedSeqOptimized.scala:51)
	at scala.collection.IndexedSeqOptimized$class.foldLeft(IndexedSeqOptimized.scala:60)
	at scala.collection.mutable.WrappedArray.foldLeft(WrappedArray.scala:34)
	at scala.collection.TraversableOnce$class.$div$colon(TraversableOnce.scala:138)
	at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:105)
	at sbt.Load$.sbt$Load$$expandSettings$1(Load.scala:682)
	at sbt.Load$.resolveProject(Load.scala:684)
	at sbt.Load$.finalizeProject$1(Load.scala:549)
	at sbt.Load$.discoverAndLoad$1(Load.scala:562)
	at sbt.Load$.loadTransitive(Load.scala:570)
	at sbt.Load$.loadProjects$1(Load.scala:442)
	at sbt.Load$.loadUnit(Load.scala:446)
	at sbt.Load$$anonfun$18$$anonfun$apply$11.apply(Load.scala:281)
	at sbt.Load$$anonfun$18$$anonfun$apply$11.apply(Load.scala:281)
	at sbt.BuildLoader$$anonfun$componentLoader$1$$anonfun$apply$4$$anonfun$apply$5$$anonfun$apply$6.apply(BuildLoader.scala:91)
	at sbt.BuildLoader$$anonfun$componentLoader$1$$anonfun$apply$4$$anonfun$apply$5$$anonfun$apply$6.apply(BuildLoader.scala:90)
	at sbt.BuildLoader.apply(BuildLoader.scala:140)
	at sbt.Load$.loadAll(Load.scala:334)
	at sbt.Load$.loadURI(Load.scala:289)
	at sbt.Load$.load(Load.scala:285)
	at sbt.Load$.load(Load.scala:276)
	at sbt.Load$.apply(Load.scala:130)
	at sbt.Load$.defaultLoad(Load.scala:36)
	at sbt.BuiltinCommands$.doLoadProject(Main.scala:481)
	at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:475)
	at sbt.BuiltinCommands$$anonfun$loadProjectImpl$2.apply(Main.scala:475)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$1$$anonfun$apply$2.apply(Command.scala:59)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$$anonfun$applyEffect$2$$anonfun$apply$3.apply(Command.scala:61)
	at sbt.Command$.process(Command.scala:93)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:98)
	at sbt.MainLoop$$anonfun$1$$anonfun$apply$1.apply(MainLoop.scala:98)
	at sbt.State$$anon$1.process(State.scala:184)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:98)
	at sbt.MainLoop$$anonfun$1.apply(MainLoop.scala:98)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.MainLoop$.next(MainLoop.scala:98)
	at sbt.MainLoop$.run(MainLoop.scala:91)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:70)
	at sbt.MainLoop$$anonfun$runWithNewLog$1.apply(MainLoop.scala:65)
	at sbt.Using.apply(Using.scala:24)
	at sbt.MainLoop$.runWithNewLog(MainLoop.scala:65)
	at sbt.MainLoop$.runAndClearLast(MainLoop.scala:48)
	at sbt.MainLoop$.runLoggedLoop(MainLoop.scala:32)
	at sbt.MainLoop$.runLogged(MainLoop.scala:24)
	at sbt.StandardMain$.runManaged(Main.scala:53)
	at sbt.xMain.run(Main.scala:28)
	at xsbt.boot.Launch$$anonfun$run$1.apply(Launch.scala:109)
	at xsbt.boot.Launch$.withContextLoader(Launch.scala:128)
	at xsbt.boot.Launch$.run(Launch.scala:109)
	at xsbt.boot.Launch$$anonfun$apply$1.apply(Launch.scala:35)
	at xsbt.boot.Launch$.launch(Launch.scala:117)
	at xsbt.boot.Launch$.apply(Launch.scala:18)
	at xsbt.boot.Boot$.runImpl(Boot.scala:56)
	at xsbt.boot.Boot$.main(Boot.scala:18)
	at xsbt.boot.Boot.main(Boot.scala)
[error] java.lang.NoSuchMethodError: sbt.Def$.settings(Lscala/collection/Seq;)Lscala/collection/Seq;```

**Comment by marcospereira** [comment]
Hi @dmurvihill,

Twirl 1.3.x was released targeting Play 2.6.x which [requires at least sbt 0.13.15](https://www.playframework.com/documentation/2.6.x/Migration26#sbt-upgrade-to-0.13.15). I saw the readme for Twirl is now incorrect and says it requires sbt 0.13.5 or higher. The reality is that it requires a newer version just like Play 2.6.

You may want to try Twirl 1.2.0 if you need support for older versions of sbt.

Given all that, I'm closing this issue since we will not support older versions of sbt in 1.3.x.

Best.

**Comment by dmurvihill** [comment]
Thanks Marco!
